src/frontend/planner_test/tests/testdata/output/generated_columns.yaml

```diff
@@ -4,11 +4,12 @@
     explain create table t1 (v1 int as v2-1, v2 int, v3 int as v2+1);
   explain_output: |
     StreamMaterialize { columns: [v1, v2, v3, _row_id(hidden)], stream_key: [_row_id], pk_columns: [_row_id], pk_conflict: Overwrite }
-    └─StreamExchange { dist: HashShard(_row_id) }
-      └─StreamRowIdGen { row_id_index: 3 }
-        └─StreamProject { exprs: [(v2 - 1:Int32) as $expr1, v2, (v2 + 1:Int32) as $expr2, _row_id] }
-          └─StreamDml { columns: [v2, _row_id] }
-            └─StreamSource
+    └─StreamRowIdGen { row_id_index: 3 }
+      └─StreamUnion { all: true }
+        └─StreamExchange { dist: HashShard(_row_id) }
```

Comment:
How can we exchange by `HardShard(_row_id)` before `RowIdGen`? What's the values of the `_row_id` column the exchange see?

Reply:
when DML executor handle a Update/delete, the there is a row_id value, and when an insert statement processed, the row_id will be None to wait be assigned by the RowIdGen